locales/uk_UA/LC_MESSAGES/duckduckgo.po

```diff
@@ -173,7 +171,7 @@ msgstr "Реклама недоречна"
 
 msgctxt "feedback form"
 msgid "Ad is irrelevant"
-msgstr "Реклама не релевантна"
+msgstr "Реклама невідповідна"
```

Comment:
This reverse-translates to `inappropriate` rather than `irrelevant`. The previous string has `msgid "Ad is inappropriate"`, so maybe this is a mistake?